docs/code-of-conduct-response.rst

```diff
@@ -0,0 +1,58 @@
+
+If the violation occurs at an event (conference, meetup), get in touch with one of the event staff right away, in any of the following ways:
+
+* In person
+* By phone
+* By email
+
+In your report please include:
+
+* Your contact information (so we can get in touch with you if we need to follow up)
+* Names of any individuals involved (real names, nicknames, or pseudonyms). If there were other witnesses besides you, please try to include them as well.
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. If there is a publicly available record (e.g. a mailing list archive or a public chat logger) please include a link.
+* Any extra context you believe existed for the incident.
+* Whether you believe this incident is ongoing.
+* Any other information you believe we should have.
+
+What happens after you file a report
+------------------------------------
+
+You will receive an email from the WTD staff acknowledging receipt. We promise to acknowledge receipt within 24 hours, and will aim for much quicker than that.
+
+The staff will meet immediately to review the incident and determine the following:
+
+* What happened.
+* Whether this event constitutes a code of conduct violation.
+* Who the bad actor was.
+* Whether this is an ongoing situation, or if there is a threat to anyone's physical safety.
+* If this is determined to be an ongoing incident or a threat to physical safety, the staff's immediate priority will be to protect everyone involved. This means we may delay an "official" response until we believe that the situation has ended and that everyone is physically safe.
+
+After the staff has a complete account of the events they will make a decision as to how to response. Responses may include:
+
+* Nothing (if we determine no violation occurred).
+* A private reprimand from the staff to the individual(s) involved.
+* A public reprimand.
+* An imposed vacation (for example, asking someone to "take a week off" from Slack or the forum).
+* A permanent or temporary ban from some or all WTD spaces (Slack, the forum, meetups).
+* A request for a public or private apology.
+
+We'll respond within one week to the person who filed the report with either a resolution or an explanation of why the situation is not yet resolved.
+
+Once we've determined our final action, we'll contact the original reporter to let them know what action we'll be taking, if any. We'll take into account feedback from the reporter on the appropriateness of our response, but we don't guarantee we'll act on it.
+
+Finally, the staff may choose to make a public report of the incident.
+
+Appealing
+---------
+
+Only permanent resolutions (such as bans) may be appealed. To appeal a decision, contact the WTD staff at report_ with your appeal and they will review the case.
```

Comment:
Maybe -- I think judges sometimes reverse their own rulings on appeal (after sleeping on it)